CefSharp.Wpf/DragCursorProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Windows.Input;
+using System.Windows.Interop;
+using CefSharp.Enums;
+using Microsoft.Win32.SafeHandles;
+
+namespace CefSharp.Wpf 
+{
+
+    internal static class DragCursorProvider 
+    {
+        [DllImport("kernel32.dll")]
+        private static extern IntPtr LoadLibrary(string dllToLoad);
+
+        [DllImport("user32.dll")]
+        private static extern IntPtr LoadCursor(IntPtr hInstance, ushort lpCursorName);
+        
+        private static readonly Dictionary<DragOperationsMask, Cursor> DragCursors;
+
+        static DragCursorProvider() 
+        {
+            var library = LoadLibrary("ole32.dll");
+            DragCursors = new Dictionary<DragOperationsMask, Cursor>()
+            {
+                { DragOperationsMask.None, GetCursorFromLib(library, 1) },
+                { DragOperationsMask.Move, GetCursorFromLib(library, 2) },
```

Comment:
Is `Move` substantially different from `SizeAll`? Be better to use the built in cursor where possible.

https://docs.microsoft.com/en-us/dotnet/api/system.windows.input.cursors.sizeall?view=netframework-4.7.2#System_Windows_Input_Cursors_SizeAll

Reply:
Drag&drop cursors don't exist in the built-in collection.

These are the ones we get from ole32:
https://i.stack.imgur.com/TffLL.png